### PR TITLE
fix(kucoin) - auto migration detect

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

File: ts/src/kucoin.ts [hunk 1]
```diff
@@ -1227,14 +1228,13 @@ export default class kucoin extends Exchange {
```
Comment (carlosmiei) [inline ts/src/kucoin.ts:1228]: @ttodua options use `hf` but your methods uses `hfMigrated` I would rather use `hf` only

Comment (ttodua) [reply ts/src/kucoin.ts:1228]: right, done

Comment (ttodua) [reply ts/src/kucoin.ts:1228]: oh, static tests fail now, i'll handle that and ping you

**Comment by ttodua** [comment]
@carlosmiei this passes now

**Comment by carlosmiei** [comment]
@ttodua can you apply the suggestions discussed yesterday here pls

**Comment by ttodua** [comment]
@carlosmiei the other than what's pushed few minutes ago?

**Comment by carlosmiei** [comment]
@ttodua Sorry my page was outdaded and haven't seen the latest commit

**Pull request merged**